src/sentry/search/utils.py

```diff
@@ -190,7 +223,9 @@ def get_date_params(value, from_field, to_field):
     'app',
     'os.name',
     'url',
-    'event.timestamp'])
+    'event.timestamp'
+    'timesSeen',
```

Comment:
Not 100% sure what is considered stylistically consistent/preferred here at this point.